modules/howtos/pages/distributed-acid-transactions-from-the-sdk.adoc

```diff
@@ -1,50 +1,52 @@
-= Distributed Transactions from the Java SDK
-:description: A practical guide to using Couchbase’s distributed ACID transactions, via the Java API.
-:navtitle: ACID Transactions
+= Distributed ACID Transactions from the Java SDK
+:description: A practical guide on using Couchbase Distributed ACID transactions, via the Java SDK.
 :page-partial:
 :page-topic-type: howto
 :page-aliases: acid-transactions
 :page-toclevels: 2
+:page-pagination: next
 
 include::project-docs:partial$attributes.adoc[]
 include::partial$acid-transactions-attributes.adoc[]
 
 [abstract]
 {description}
 
-include::{version-server}@sdk:shared:partial$acid-transactions.adoc[tag=intro]
+IMPORTANT: The Couchbase transactions feature was previously distributed as a separate library, before being integrated into the SDK from version `3.3.0`.
+Users of the transactions library are recommended to follow the xref:project-docs:distributed-acid-transactions-migration-guide.adoc[Transactions Migration Guide].
+
+This guide will show you examples of how to perform multi-document ACID (atomic, consistent, isolated, and durable) database transactions within your application, using the Couchbase Java SDK.
+
+Refer to the xref:concept-docs:transactions.adoc[Distributed ACID Transactions] concept material for a high-level overview.
+
+== Prerequisites
+
+[{tabs}]
+====
+Couchbase Capella::
++
+--
+* Couchbase Capella account.
+* You should know how to perform xref:howtos:kv-operations.adoc[key-value] or xref:howtos:n1ql-queries-with-sdk.adoc[query] operations with the SDK.
+* Your application should have the relevant roles and permissions on the required buckets, scopes, and collections, to perform transactional operations.
+Refer to the xref:cloud:organizations:organization-projects-overview.adoc[Organizations & Access] page for more details.
+* If your application is using xref:concept-docs:xattr.adoc[extended attributes (XATTRs)], you should avoid using the XATTR field `txn` -- this is reserved for Couchbase use.
+--
+
+Couchbase Server::
++
+--
+* Couchbase Server (6.6.1 or above).
+* You should know how to perform xref:howtos:kv-operations.adoc[key-value] or xref:howtos:n1ql-queries-with-sdk.adoc[query] operations with the SDK.
+* Your application should have the relevant roles and permissions on the required buckets, scopes, and collections, to perform transactional operations.
+Refer to the xref:{version-server}@server:learn:security/roles.adoc[Roles] page for more details.
+* If your application is using xref:concept-docs:xattr.adoc[extended attributes (XATTRs)], you should avoid using the XATTR field `txn` -- this is reserved for Couchbase use.
+* NTP should be configured so nodes of the Couchbase cluster are in sync with time.
```

Comment:
Note that I added this **NTP** bit only for Couchbase Server, not entirely sure if you need to do this in Capella also?

@RichardSmedley do you have any idea?

Reply:
AWS do provide NTP, and I'd be very surprised if we'd turned it off.
(But worth checking to be sure)